locations/philmel.yml

```diff
@@ -9,11 +9,19 @@ hosts:
   - hostname: philmel-core
     role: corerouter
     model: "linksys_e8450-ubi"
-    wireless_profile: freifunk_default
-  - hostname: philmel-nord-m2
+  - hostname: philmel-o-nf-5ghz
+    role: ap
+    model: "mikrotik_sxtsq-5-ac"
+    mac_override:
+      # TODO: correct this mac address
+      eth0: cc:2d:e0:9c:4f:00
+  - hostname: philmel-s-nf-5ghz
     role: ap
-    model: "ubnt_nanostation-m2_xm"
+    model: "mikrotik_sxtsq-5-ac"
     wireless_profile: freifunk_default
+    mac_override:
+      # TODO: correct this mac address
+      eth0: cc:2d:e0:9c:4f:00
```

Comment:
Using the same mac-adress twice within the same local network might cause some issues i think. Maybe giving it another random one until the right ones are set could be a fix for this problem.